typings/sharify.d.ts

```diff
@@ -25,6 +25,7 @@ declare module "sharify" {
       readonly GEMINI_CLOUDFRONT_URL: string
       readonly GENOME_URL: string
       readonly GOOGLE_ADWORDS_ID: string
+      readonly HASHTAG_LAB_ADS_ALLOWLIST: string // TODO: Remove after externally served ads are implemented
```

Comment:
So this doesn't break Storybooks (😉 😉) this also needs to be added to the Webpack config [here](https://github.com/artsy/reaction/blob/master/.storybook/webpack.config.js#L18-L62).